content/blog/2023/fastapi-path-operations-for-django-developers.md

```diff
@@ -0,0 +1,258 @@
+---
+title: FastAPI Path Operations for Django Developers
+author: ed
+date: 2023-10-24
+tags:
+  - Article
+  - Python
+  - Django
+  - FastAPI
+image:
+  src: blog/2023/fast.jpg
+summary: |
+  FastAPI path operations are the equivalent to Django views. In this article we
+  explore the differences, advantages, and gotchas of using them from the
+  perspective of a Django developer.
+---
+
+You might have heard about FastAPI, a modern and fast web framework for building
+APIs with Python. You might be wondering how it compares to Django, the most
+popular and mature web framework for Python. In this series, I will try to
+answer this question by comparing Django and FastAPI on various aspects and
+features after we successfully converted an internal project from Django to
+FastAPI.
+
+1. FastAPI path operations for Django Developers (this article)
+2. SQLAlchemy for Django Developers (coming soon)
+3. Testing a FastAPI application (coming soon)
+4. How to use FastAPI dependency injection everywhere (coming soon)
+
+## Why is FastAPI worth considering?
+
+*This section contains opinions. Skip to [Django Views](#django-views) for the
+actual comparison.*
```

Comment:
I'd probably remove this -- opinions are helpful, and the title of this section already gives the reader a heads-up.